contracts/NFT/OGFoundersRootstockCollective.sol

```diff
@@ -24,19 +25,37 @@ contract OGFoundersRootstockCollective is ERC721NonTransferrableUpgradable {
   address public stRIF;
   uint256 public firstProposalDate;
   // Counter for the total number of minted tokens
-  uint8 private _totalMinted;
+  uint256 private _totalMinted;
```

Comment:
Why have you changed for uint256? What's the purpose? It will never get past 256, cause the total is 150 and for this collection that's it.

Reply:
I was under the impression that if you use uint8 in isolation the compiler will automatically convert it to uint256. The way to prevent this would be to have enough uint8s to cover the entire slot. That being said, not the time to debate this.

Reply:
Because now it’s possible to adjust the token quantity using the setIpfFOlder function. The gas savings here are minimal and matter far less than the potential overflow that could occur when adding new tokens, where the uint8 array may not be large enough to accommodate everything.